src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/SupportController.java

```diff
@@ -74,7 +74,7 @@ public void resendStaffNotifications(
             .orElseThrow(() -> new NotFoundException(CLAIM + referenceNumber + " does not exist"));
 
         switch (event) {
-            case "claim-issued":
+            case "claim-submitted-post-payment":
```

Comment:
Please change back the event behaviour hasn't changed, its still when we issue the claim

Reply:
@dharmendrak please revert event rename. Event name in context of SupportController should be simple and reflect what happened. The fact that case is in CCD after successful payment was made does not matter.